### ISSUE TITLE
Write CitySource blurb for Alpha.austin.gov

### ISSUE THREAD

**Issue by aineeathar** [cityofaustin/techstack#60]
ON HOLD until clearance from Doug / Marni

**Comment by cthibaultatx** [comment]
@aineeathar HR writes citysource, right?

**Comment by aineeathar** [comment]
Hmm not sure but CPIO asked us to put something in there for the alpha launch. Anyone can submit something.
________________________________

**Comment by ashleecoa** [comment]
@cthibaultatx @aineeathar Joan in my department (CPIO) writes Citysource

**Comment by wilhitem** [comment]
Hey ainee, can you provide more details in this....i don't want any of us jumping to conclusions. My interpretation is that the goal from CPIO would be 'As a city employee, I want to be updated on progress and goals for the alpha.austin.gov so that I have a better understanding of the path and vision of our team as it supports the city.'

Is that accurate? Do you have other details? Is this tied to the audience matrix and details that might be in there already? It would be great to link to that or other supporting documents so that so all relevant info is available and we make sure we prioritize/scope the time commitment effectively.

Thank you lady!

**Comment by aineeathar** [comment]
We haven't written it yet- I'm still waiting on info from Gina on things like how long it has to be and I will def send it for you to review. They just asked to hold until the content was live. It's come up in the meetings and is in the meeting notes.


Honestly no one really is uses the collateral docs so the meeting notes from the comms meetings are the most accurate resources.


Your articulation of the goal is spot on: the audience is general City staff, just to keep them updated.


Ainee Athar

Design Technology and Innovation Fellows

Community Engagement and Communications

________________________________

**Comment by aineeathar** [comment]
I wrote a draft today. It must be submitted by next Thurs at noon to go out by the following Tuesday.
Here's a link. Following up with a separate email to get edits and feedback from Doug, Marni, Adam, etc.


https://docs.google.com/document/d/1aSpwNS1cpEbo8FvlysrUdtC_4w0gMWFOcde5eLVTulA/edit

**Comment by LauraTrujillo** [comment]
@amenity Should I reassign this to myself and keep it in the backlog until we get the greenlight?

**Comment by amenity** [comment]
@LauraTrujillo - that would be great. @succarwagner - do you have a sense of when we will want to write about the Alpha site in City Source?

**Comment by wilhitem** [comment]
@amenity it's probably not the best time for us to send out something along the context of what Ainee wrote...there's no call to action and we definitely want people to be excited, but might want to narrow in on something more meaty if they aren't signing up to participate in some capacity.

@LauraTrujillo and the content team were talking about a city-wide funshop...maybe check with them as to whether they are ready for that to be publicized/have a sign up page or some strategy around it. Want to check in with them? 

something you and i can prioritize is reviewing the communication strategy w/r/t the milestones we might want to publicize and review with team/stakeholders - happy to jump on that next week

**Comment by LauraTrujillo** [comment]
@wilhitem Agreed! I'm checking to see how much space (i.e. seating) we have for a city-wide funshop before we turn on the funshop signal :) First one could probably be held at the end of this month. I'll figure out the logistics and then check in with you and @amenity about a CitySource post.

**Comment by LauraTrujillo** [comment]
sounds like a plan, @succarwagner!
